docs/wordpress.md

```diff
@@ -93,7 +93,7 @@ database containers. If you're using [Docker Machine](https://docs.docker.com/ma
 
 ## More Compose documentation
 
-- [User guide](/)
+- [User guide](index.md)
```

Comment:
yes, very interesting indeed.

I'm going to investigate, but the fix may happen after I've enabled all the jenkins jobs for all the repos.

For now, I'd suggest merging this PR, so that any other Doc PRs don't fail validation.